Blur Animation source code and simplified output (#328)
datacarpentry/image-processing: episodes/06-blurring.md

::::::::::::::::::::::::::::::::::::::::::::::::::

> Let's consider a very simple image to see blurring in action. The animation below shows how the blur kernel (large red square) moves along the image on the left in order to calculate the corresponding values for the blurred image (yellow central square) on the right. In this simple case, the original image is single-channel, but blurring would work likewise on a multi-channel image.

Reviewer: ```suggestion
Let's consider a very simple image to see blurring in action. The animation below shows how the blur kernel (large red square) moves along the image on the LHS in order to calculate the corresponding values for the blurred image (yellow central square) on the RHS. In this simple case, the original image is single-channel, but blurring would work likewise on a multi-channel image.
```

Author: I will keep the same 'on the left' and 'on the right' language as in `create_blur_animation.py`

> Let's take a very simple grayscale image to see blurring in action. The animation below shows how the blur kernel (large red square) moves along the image on the left in order to calculate the corresponding values for the blurred image (yellow central square) on the right. In this simple case the image is composed of only a single channel, but it would work as well on a multi-channel image.

Reviewer: ```suggestion
Let's consider a very simple image to see blurring in action. The animation below shows how the blur kernel (large red square) moves along the image on the left in order to calculate the corresponding values for the blurred image (yellow central square) on the right. In this simple case, the original image is single-channel, but blurring would work likewise on a multi-channel image.
```
I suggested removing 'grayscale' not because it is wrong, of course, but just to avoid the repetition afterwards (when we mention it is single-channel) and because it's actually even 'simpler' than grayscale, it's binary.

Author: Agreed, thanks for the rephrasing. I implemented this change in the next commit.
